packages/renderer/tests/setupTests.ts

```diff
@@ -0,0 +1,50 @@
+import { afterEach, vi } from 'vitest';
+
+// Mock preload modules
+vi.mock('#preload', () => ({
```

Comment:
is it a good idea to do this dynamically? like get the keys recursively and apply the mock fn?

Reply:
totally! mock updated